src/error.rs

```diff
@@ -13,6 +14,8 @@ pub enum Error {
     SerdeJson(#[cause] serde_json::Error),
     #[fail(display = "{}", _0)]
     SerdeToml(#[cause] toml::de::Error),
+    #[fail(display = "Acquiring lock failed")]
```

Comment:
Then just change this to:
`#[fail(display = "{}", _0)]`

Which basically says if this error happens for it's `Display` implementation grab the first value (the `_0`) and use it's display implementation for this string. Since `PoisonError` implements `std::error::Error` we know it has a display implementation which we want to use here. same reason for the `#[cause]` part